packages/plugins/openapi/tests/openapi-restful.test.ts

```diff
@@ -55,12 +56,21 @@ model post_Item {
     published Boolean @default(false)
     viewCount Int @default(0)
     notes String?
+    likes PostLike[]
 
     @@openapi.meta({
         tagDescription: 'Post-related operations'
     })
 }
 
+model PostLike {
```

Comment:
Just curious, do you know why the "PostLike" model doesn't show up as top level in the updated baseline files?

Reply:
Hmm no. Could be that this requires some more work. I'll look into it.

Reply:
I see. That makes sense. Thank you!